src/MegaPoker.sol

```diff
@@ -61,6 +62,7 @@ contract MegaPoker is PokingAddresses {
         (ok,) = univ2wbtcdai.call(abi.encodeWithSelector(0x18178358));
         (ok,) = matic.call(abi.encodeWithSelector(0x18178358));
         (ok,) = wsteth.call(abi.encodeWithSelector(0x18178358));
+        (ok,) = crvv1ethsteth.call(abi.encodeWithSelector(0x18178358));
```

Comment:
I've pinged OCU about this. We might be able to get away with poking this once a day like the GUNI tokens.

Reply:
nvm price is in fiat :sweat_smile: